src/components/ContentSidebar/ActivityFeed/comment/Comment.js

```diff
@@ -84,7 +79,9 @@ class Comment extends React.Component<Props, State> {
     approvalCommentFormSubmitHandler = (): void => this.setState({ isInputOpen: false, isEditing: false });
     updateTaskHandler = (args: any): void => {
         const { onEdit } = this.props;
-        onEdit(args);
+        if (onEdit) {
```

Comment:
If you wanna remove the `if`, you could do `const { onEdit = noop } = this.props;`

Your call!